apps/web/pages/v2/success.tsx

```diff
@@ -141,7 +141,15 @@ type SuccessProps = inferSSRProps<typeof getServerSideProps>;
 export default function Success(props: SuccessProps) {
   const { t } = useLocale();
   const router = useRouter();
-  const { location: _location, name, reschedule, listingStatus, status, isSuccessBookingPage } = router.query;
+  const {
+    location: _location,
+    name,
+    email,
+    reschedule,
+    listingStatus,
+    status,
+    isSuccessBookingPage,
+  } = router.query;
```

Comment:
Since we're depending so much on `router.query` we should define a zod schema and parse it.

Reply:
@zomars since this issue is open #5053, can we skip this for now?

Reply:
Sure